packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -230,6 +258,28 @@ function Directions({ isOpen, onBack, onSetSize, snapPointSwiped, onRouteFinishe
         setActiveStep(0);
         setSubstepsOpen(false);
         setSize(snapPoints.FIT);
+
+        // Go to current venue once directions are 'Closed'
+        if (isDesktop) {
+            goTo(currentVenue.geometry, mapsIndoorsInstance, 0, 0, getZoomLevel(startZoomLevel), pitch, bearing);
+        } else {
+            getMobilePaddingBottom().then(mobilePaddingBottom => {
+                goTo(currentVenue.geometry, mapsIndoorsInstance, mobilePaddingBottom, 0, getZoomLevel(startZoomLevel), pitch, bearing);
+            });
+        }
+    }
```

Comment:
Are these changes necessary in the context of the issue with minZoom and Wayfinding?